docker/images/pinot/bin/generator.sh

```diff
@@ -18,7 +18,7 @@
 # under the License.
 #
 
-JAR_PATH="/opt/pinot/lib/pinot-all-0.3.0-SNAPSHOT-jar-with-dependencies.jar"
+JAR_PATH="/opt/pinot/lib/pinot-all-0.4.0-SNAPSHOT-jar-with-dependencies.jar"
```

Comment:
Can we use * to match the version like pinot-all-*.jar ? So we don’t need to bother upgrade the version anymore.

Reply:
ok. had to use **find** but should work for any version now